### DISCUSSION TITLE
Use tailwind theme variables such as primary, secondary, ring, etc instead of hardcoded colors?

### DISCUSSION THREAD

**Discussion by pgupta1795** [alibey-10/indie-ui#26] [Ideas]
Hi,
Components looks really great and reusable.
Thank You for this.
Is it possible to refactor and use the tailwind theme variables such as primary, secondary, ring, etc instead of hardcoded colors?

**Comment by alibey-10** [comment]
Sorry for the late response. @pgupta1795 
afaik, Tailwind doesn't suggest any set of theme variables. Most components are pretty basic and easy to customize.
